project/AppDependencies.scala

```diff
@@ -7,7 +7,9 @@ object AppDependencies {
   val compile = Seq(
     "uk.gov.hmrc" %% "bootstrap-backend-play-30" % bootstrapVersion,
     "com.beachape" %% "enumeratum" % "1.7.3",
-    "com.beachape" %% "enumeratum-play-json" % "1.8.0"
+    "com.beachape" %% "enumeratum-play-json" % "1.8.0",
```

Comment:
I hadn't noticed this before - but where does this library come from and is it used elsewhere in HMRC? We have to be cautious adding new libraries, we may need to have approval.